new-files

SaadMahi opened the pull request.

SaadMahi committed:
new-files

ManSOorcode commented on style.css:
```diff
@@ -0,0 +1,101 @@
+  --main-radius: 5px;
+  --main-padding: 5px;
+}
+
```
add this in your css :
```js
*{
  margin:0;
  padding:0;
  box-sizing:border-box;
};
```

SaadMahi replied:
i would like to know in detail what is this used for?

SaadMahi replied:
added

ManSOorcode commented on style.css:
```diff
@@ -0,0 +1,101 @@
+  font-size: 0.75em;
+
+  text-align: center;
+
```
add this in your css:
```js
padding: var(--main-padding);
```
so this will remove the scroll bar which come on your screen

SaadMahi replied:
padding: var(--main-padding); added

ManSOorcode reviewed (changes requested):
done this above change and it will remove the scroll bar from your grid project

murtazshaikh commented on index.html:
```diff
@@ -0,0 +1,27 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
set title according to project name. for example: <title>Responsive Grid Layout</title>

murtazshaikh commented on index.html:
```diff
@@ -0,0 +1,27 @@
+  </head>
+  <body>
+    <div class="container">
+      <nav>NavBar</nav>
```
Read more about semantic tags: https://www.semrush.com/blog/semantic-html5-guide/#html-semantic-tags-for-structure

SaadMahi replied:
great! have saved the link will definitely go through it.

SaadMahi committed:
Updated style.css

added padding: var(--main-padding); to eliminate the scroll bar

SaadMahi committed:
Updated index.html

Given Title name as [ Responsive Grid Layout ]

SaadMahi reviewed (commented):
CSS resolution pending in which we use universal tag *.

SaadMahi committed:
css style updated

added *{ margin:0; padding:0; box-sizing:border-box; }; at top

SaadMahi committed:
style.css

universal tags with properties add below line 5 as suggested by Mansoor brother

SaadMahi reviewed (commented):
new-changes made to style.css. Those are using universal selector below root:

SaadMahi committed:
Update style.css

murtazshaikh reviewed (approved):
Great @SaadMahi

SaadMahi commented:
> Great @SaadMahi

looks like page isn't mobile responsive any more due to the above changes

SaadMahi commented:
> Great @SaadMahi

Kindly view page deployment, the page isn't displaying as desired, i think it's due to the use of universal selector.

SaadMahi committed:
style.css 

* universal selector removed due to webpage display error

SaadMahi committed:
style.css

removed padding from line 22, as it was disturbing the display of webpage

SaadMahi committed:
Update style.css

SaadMahi committed:
removed scroll bar by suing * selector

SaadMahi committed:
Merge branch 'b2' of https://github.com/SaadMahi/11-Responsive-Css-Grid into b2

SaadMahi committed:
added padding and border box as murtaza shaikh directd

ManSOorcode reviewed (approved):
LGTM
